prac_05/emails.py

```diff
@@ -0,0 +1,15 @@
+email = input("Email: ")
+print(email)
+name_and_email = {}
```

Comment:
Using "name_to_email" for the dictionary name could better fit naming conventions?